static/new_home/style.css

```diff
@@ -631,6 +631,14 @@ header
   color: #eb5d1e;
 }
 
+@media screen and (min-width: 768px)
+  {
+    .col-md-6{
```

Comment:
> First of all, really sorry for the slow review!
> 
> I still think there's a need to decrease the paddings inside the cards and the margin outside it. Although the cards are sized well now (good work for that), it doesn't look good for screen-width > 768px, does it? And I think it will look fine after reducing the margins and paddings wherever necessary.
> 
> For screen-width < 768px, overlap the bottom card over the top (for 10 px), don't keep them separate.


@garg3133 By overlapping the cards for width < 768px, do you mean this?
![Screenshot_1](https://user-images.githubusercontent.com/61618767/114035060-644cb180-989c-11eb-81d8-cd29ffb32793.png)

Reply:
Yes, I meant like this. But the second card looks really odd with no heading. I think we should have some heading on this card, like "Write to us", "Send us a message" or something else that'll feel right in this case.

Reply:
Also, the card looks fine without heading when they are side-by-side. So, I think we can just display the heading on small screens and hide it on larger screens.

Reply:
Sure, I'll add that. Should I write something like "Email us" or "Get in touch" or maybe something like "Let's start a conversation"?

Reply:
"Get in touch" sounds better!

Reply:
Please check this, I reduced the paddings and the margins inside the cards, so now it looks like this.



Original             |  Updated
:-------------------------:|:-------------------------:
![Screenshot_3](https://user-images.githubusercontent.com/61618767/114045901-03c27200-98a6-11eb-9919-54a62fcfa298.png) | ![Screenshot_4](https://user-images.githubusercontent.com/61618767/114045930-09b85300-98a6-11eb-9187-f496a6d8406b.png)

Reply:
Here are a few suggestions:
* Retain the gap below Jagrati.
* Reduce the margin around the two cards (`containerly` class' margins).
* Reduce the padding on the left side of the location, call and email icons (or let it be as it is if it looks fine after reducing the margins in the last step).
* Increase margins a bit around the contact form inside the right card.


You're making the changes for width > 768px only, right? Keep it for widths >=768px and <1200px.

Reply:
The margins around the two cards(`containerly` class) is currently specified to auto. How much should I reduce it??

Reply:
`containerly` class must be having a `width` property then. Increase that to `95%` or whatever looks good.

Reply:
One more thing, can you add a property `margin-left: -10px` to the right card for screen widths > 768px ?

Reply:
> `containerly` class must be having a `width` property then. Increase that to `90%` - `95%`, whatever looks good.

Should I only increase this width for screen size between 1200 and 768px  or more than 768px

Reply:
> One more thing, can you add a property `margin-left: -10px` to the right card for screen widths > 768px ?

Sure I'll add that

Reply:
### After doing all the above said changes it now looks like this. Is this alright?

https://user-images.githubusercontent.com/61618767/114267360-b61a4680-9a18-11eb-9032-d3909b3d1929.mp4

Reply:
> Should I only increase this width for screen size between 1200 and 768px or more than 768px

1200 to 768px

> After doing all the above said changes it now looks like this. Is this alright?

Yeah, it looks great, except for the range between 900 px and 768px (where connect with us icons get into two lines). I can't think of a fix for this other than putting the cards in rows for <900px itself.

Reply:
I fixed that by just decreasing the margins. Here's how it looks now 

https://user-images.githubusercontent.com/61618767/114294150-26c56f80-9aba-11eb-989c-ff70b8d39ca4.mp4

Reply:
Let me know if I should commit my changes now.

Reply:
Yeah, it looks much better now! Sure, you can commit your changes.

Reply:
I've committed my changes. Let me know if there's anything else to be done.